alerting/src/main/resources/com/amazon/opendistroforelasticsearch/alerting/com.amazon.opendistroforelasticsearch.alerting.txt

```diff
@@ -22,6 +30,16 @@ class com.amazon.opendistroforelasticsearch.alerting.script.TriggerExecutionCont
     Exception getError()
 }
 
+class com.amazon.opendistroforelasticsearch.alerting.script.AggregationTriggerExecutionContext {
```

Comment:
Does this and `AggregationTrigger` need to be defined here? This is a definition of classes used by Painless which we won't be using for Aggregation Monitors.

Reply:
Good catch, i think i made changes to the TriggerScript before to add support for Aggregation monitor but ended up removing it later.